### ISSUE TITLE
Invariant violation: Native component for 'AirMapMarker' does not exist

### ISSUE THREAD

**Issue by sami718** [react-community/react-native-maps#1983]
When I was using **react-native-maps@0.15.2** it was working fine. Now when I upgraded it to **react-native-maps@0.19.0** it shows following error -  native component AirMapMarkerdoes not exist
I have read installation document and installed it properly.
The react, react-native, react-native-maps versions-

   ```
 react-native": 0.52.0
    react: 16.2.0
    react-native-maps: 0.19.0 
```
![screen shot 2018-01-22 at 11 10 13 am](https://user-images.githubusercontent.com/28917467/35206899-ba26a396-ff65-11e7-9679-91d6de20909c.png)

**Comment by alvelig** [comment]
Can you please make a reproducible demo forking https://github.com/alvelig/react-native-maps-example. I suppose that react-native link is missing

**Comment by bilwani101** [comment]
Experiencing the same issue. But works fine after following the below link
https://codeburst.io/react-native-google-map-with-react-native-maps-572e3d3eee14

**Comment by MarkOSullivan94** [comment]
Currently having the same issue on Android. Anyone got a solution for Android? Nearly every solution I see is for iOS.

**Comment by rborn** [comment]
Closing as we start to enforce the issue template. Sorry.

**Comment by MarkOSullivan94** [comment]
@rborn why did you not ping the creator of this issue before closing it? They could have updated their original post.

**Comment by rborn** [comment]
@MarkOSullivan94 the issue template specifies that if the issue doesn't have the template we close it right from the start. We didn't enforce this right away so there was time to update or adopt it for new issues.

This issue in particular might have been here before the template, but we are trying to keep the number if issues as small as possible, so some issues might get closed in an "unfair" way. 

However in this case @alvelig asked for more info and got no reply after 6 weeks when I closed the issue. 
We want to help, but we do this in our free time so we have a limited bandwidth 😔

I hope I answered your question.
